.github/scripts/git-get-shared-history.sh

```diff
@@ -20,6 +20,13 @@ set -euxo pipefail
 
 # We can always use some more history
 git -c protocol.version=2 fetch --deepen=100
+
+# Find out if the PR is coming from a fork
+base_clone_url=$(jq '.pull_request.base.repo.clone_url' $GITHUB_EVENT_PATH)
```

Comment:
shouldn't this be 
```suggestion
base_clone_url=$(jq '.pull_request.head.repo.clone_url' $GITHUB_EVENT_PATH)
```

to find out if it's a fork?